### PR TITLE
refactor: align with docusaurus version structure

### PR DESCRIPTION
**Pull request by jakerachleff**
Moves our docs to meet docusaurus standards.

- Current docs moved from `versioned-docs/version-2.0` to the `docs/` folder
- Old docs go under `versioned-docs/version-old`

Doing this so that we can actually conform to docusaurus standards, and therefore use their instructions on how to do things (like SEO optimization for new docs vs old docs).

## NOTES FOR REVIEW
I didn't make many changes here except for:
1. moving old doc files to the `version-1.0`
2. Updating our langsmith cookbooks script we use to generate cookbooks in the navbar

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -35,18 +35,17 @@ const config = {
...
+              label: "stable",
               badge: false,
-              banner: "unmaintained",
             },
-            "2.0": {
+            "1.0": {
               label: "stable",
               banner: "none",
               badge: false,
+              path: "old",
```
Comment (efriis) [inline docusaurus.config.js:48]:
```suggestion
              label: "old",
              banner: "unmaintained",
              badge: false,
              path: "old",
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -35,18 +35,17 @@ const config = {
...
           versions: {
             current: {
-              label: "old",
-              path: "old",
+              label: "stable",
               badge: false,
-              banner: "unmaintained",
             },
-            "2.0": {
+            "1.0": {
```
Comment (efriis) [inline docusaurus.config.js:44]: might be worth renaming this "old" just to match - doesn't have to be a number string afaik

File: versions.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 [
-  "2.0"
+  "1.0"
```
Comment (efriis) [inline versions.json:2]: if renaming to old need to update this to "old" too

**Review by efriis — COMMENTED** [review]
assuming everything else is just file moves

**Review by efriis — APPROVED** [review]
for posterity might be worth listing the move part in the pr description - not a huge deal if lots of moves

docs/ -> versioned-docs/old
versioned-docs/2.0 -> docs/

etc

**Comment by jakerachleff** [comment]
> for posterity might be worth listing the move part in the pr description - not a huge deal if lots of moves
> 
> docs/ -> versioned-docs/old versioned-docs/2.0 -> docs/
> 
> etc

updated!

**Pull request merged**